### ISSUE TITLE
Checking for list in fn-body?

### ISSUE THREAD

**Issue by palquel** [tonsky/rum#41]
In `rum.clj`, there is

```
(defn- fn-body? [form]
  (and (list? form)
       (vector? (first form))))
```

Why do we need to check if `form` is a list here? The reason I ask is that I'm writing a macro that generates a `defcs` form. I can't do this because in clojure form is an ArraySeq. `form` appears to only be a list when `defcs` is expanded directly from cljs.

**Comment by tonsky** [comment]
It can be changed to `sequential?`. Care to do a PR?

**Comment by tonsky** [comment]
Fixed in 5add8dfb (0.7.0 and later)
